src/pages/registration/resource_type_tab/sub_type_forms/artistic_types/OutputRow.tsx

```diff
@@ -86,6 +85,13 @@ export const OutputRow = ({
 
   return (
     <TableRow>
+      <TableCell sx={{ width: '10rem' }}>
+        <AnnouncementTableMoveButtons
+          index={index}
```

Comment:
Det føles rart at denne tar inn `index` bare for å bestemme hva data-testid skal være, ikke minst siden den allerede har tilgang til å sette data-testid selv via props på IconButton 🤔 

Blir vel litt mindre kode av å gjøre det slik da, men syns det også blir litt mer "mystisk" kode, ved at man ikke kan skjønne hva index brukes til uten å faktisk se koden til komponenten. Hva tenker du?

```suggestion
          data-testid={dataTestId.registrationWizard.moveUpButton(index)}
```